packages/dom/src/dom.js

```diff
@@ -40,11 +28,11 @@ function isSelectionForward( selection ) {
 	/* eslint-disable no-bitwise */
 	// Compare whether anchor node precedes focus node. If focus node (where
 	// end of selection occurs) is after the anchor node, it is forward.
-	if ( position & DOCUMENT_POSITION_PRECEDING ) {
+	if ( position & anchorNode.DOCUMENT_POSITION_PRECEDING ) {
```

Comment:
you'll see this pattern a lot through this PR. Instead of using the constant hanging off `window.Node`, use the same constant which is present on anything that implements [Node](https://dom.spec.whatwg.org/#interface-node).

Reply:
Nice! There are many more instances of this across the codebase. Could they all be replaced in this PR?

Reply:
I wanted to make sure this was an acceptable approach before pushing it further. The alternative would be to use `Node.DOCUMENT_POSITION_PRECEDING`, just not as a module level variable.

Reply:
This is a great approach!